Changes to the component code to follow coffee theme (#643)
mi6/ic-design-system: src/content/structured/components/alerts/code.mdx

export const snippetsDefault = [
  {
    language: "Web component",

>     snippet: `<ic-alert heading="Coffe Blend" message="What type of coffee is this?."></ic-alert>

Reviewer: ```suggestion
    snippet: `<ic-alert heading="Coffee Blend" message="What type of coffee is this?."></ic-alert>
```

<ic-alert variant="info" heading="Did you know?" message="Coffee is a brewed drink prepared from roasted coffee beans."></ic-alert>
<ic-alert variant="error" heading="Error" message="Incorrect blend."></ic-alert>
<ic-alert variant="warning" heading="Hot Drink" message="Please be careful the drink is hot."></ic-alert>
<ic-alert variant="success" heading="Success" message="Correct blend."></ic-alert>`,
  },
  {
    language: "React",

>     snippet: `<ic-alert heading="Coffe Blend" message="What type of coffee is this?."></ic-alert>

Reviewer: ```suggestion
    snippet: `<ic-alert heading="Coffee Blend" message="What type of coffee is this?."></ic-alert>
```

  },
  {
    language: "React",

>     snippet: `<IcAlert heading="Did you kow?" message="Coffee is delicious.">

Reviewer: ```suggestion
    snippet: `<IcAlert heading="Did you know?" message="Coffee is delicious.">
```

    <IcButton slot="action" variant="secondary" appearance="dark" onClick={(ev) => console.log(ev)}>Yes</IcButton>
</IcAlert>`,
  },
];

<ComponentPreview snippets={snippetsWithAction}>

>   <IcAlert heading="Did you know?" message="Coffee is delicious.">

Reviewer: Think this might be better as something like this:
```suggestion
  <IcAlert heading="Did you know?" message="There are a new range of coffees to try.">
```

    <IcButton
      slot="action"
      variant="secondary"
      appearance="dark"
      onClick={(ev) => console.log(ev)}
    >

>       Yes

Reviewer: And then the button can say something like "View drinks"

export const snippetsTitleAbove = [
  {
    language: "Web component",
    snippet: `<ic-alert heading="What is Coffee?" message="Coffee is a brewed drink prepared from roasted coffee beans." title-above="true"></ic-alert>`,
  },
  {
    language: "React",
    snippet: `<IcAlert heading="What is Coffee?" message="Coffee is a brewed drink prepared from roasted coffee beans." titleAbove />`,
  },
];

<ComponentPreview snippets={snippetsTitleAbove}>
  <IcAlert
    heading="What is Coffee?"

>     message="TCoffee is a brewed drink prepared from roasted coffee beans."

Reviewer: ```suggestion
    message="Coffee is a brewed drink prepared from roasted coffee beans."
```

>     message="TCoffee is a brewed drink prepared from roasted coffee beans."

Reviewer: Can you make this message longer so that the title does appear above the message?

water before being filtered out, producing a cup of coffee."></ic-alert>`,
  },
  {
    language: "React",
    snippet: `<IcAlert heading="I love coffee so much this title could be paragraphs long" message="Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain flowering plants in the Coffea genus. From the coffee fruit, the seeds are separated to produce a stable, raw product: unroasted green coffee. The seeds are then roasted, a process which transforms them into a consumable product: roasted coffee, which is ground into fine particles that are typically steeped in hot
water before being filtered out, producing a cup of coffee." />`,
  },
];

<ComponentPreview snippets={snippetsResponsiveTitle}>
  <IcAlert
    heading="I love coffee so much this title could be paragraphs long"

>     message="Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain flowering plants in the Coffea genus. From the coffee fruit, the seeds are separated to produce a stable, raw product: unroasted green coffee. The seeds are then roasted, a process which transforms them into a consumable product: roasted coffee, which is ground into fine particles that are typically steeped in hot

Reviewer: This requires quite a lot of horizontal scrolling, would it be possible to shorten it a bit?
